src/main/java/ch/njol/skript/classes/data/JavaClasses.java

```diff
@@ -325,6 +326,36 @@ public String toVariableNameString(Quaternionf quaternion) {
 						return null;
 					}
 				}));
+
+		Classes.registerClass(new ClassInfo<>(UUID.class, "uuid")
+			.user("(uuid|universal(ly)? unique identifier)s?")
```

Comment:
Is the second pattern really needed?
(I personally feel like it could be taken out)